optashift-employee-rostering-gwtui/pom.xml

```diff
@@ -91,6 +101,12 @@
       <artifactId>errai-cdi-jboss</artifactId>
       <scope>runtime</scope>
     </dependency>
+    
+    <!-- GWT compiler logger -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
```

Comment:
Why not logback just like the other modules? If we end up with 2 impl's in webapp (logback and slf4j-simple) then logging is broken
If this doesn't leak to webapp (transitive dep), can we make it `<dependency>...<optional>true</></>` to be sure?

Reply:
logback appears to be broken/incompatible with the current version of the logging system; using logback causes there to be no logger due to initialization failures (in both webapp and GWT). I can remove this, although it probably better to remove logback.